generic-styles/content-common.less

```diff
@@ -364,6 +364,16 @@ blockquote {
     }
   }
 
+  [data-type="problem"] {
+    display: flex;
+    align-items: baseline;
+
+    .os-problem-container {
+      display: grid;
```

Comment:
it works for me without this line

Reply:
In every browser? For me in Chrome and Safari it doesn’t.

Reply:
Specifically the sentences that start with capital vocals, the number goes lower.

Reply:
![image](https://user-images.githubusercontent.com/25258810/88299058-2a605e80-ccbf-11ea-9ab0-43cf41ad525a.png)

Reply:
I'm starting to hate that browsers lack compatibility 
Please add a comment above this line because it's weird to see just `display: grid` in that place